File: 7-kyu/cost-of-my-ride/javascript/solution.js
const insurance = (age, size, numOfDays) => {
  const ageCharge = age < 25 ? 10 : 0;
  let dailyCharge = 50 + ageCharge;
  switch (size) {
      case 'economy':
      break;
      case 'medium':
      dailyCharge += 10;
      break;
      default:
      dailyCharge += 15;
  }
  return Math.max(numOfDays, 0) * dailyCharge;
};

File: 7-kyu/cost-of-my-ride/python/solution.py
def insurance(age, size, num_of_days):
    age_charge = 10 * (age < 25)
    size_table = {'economy': 0, 'medium': 10, 'full-size': 15}
    size_charge = size_table.get(size, 15)
    return max(num_of_days, 0) * (50 + age_charge + size_charge)
